nodejs/week1/warmup.js

```diff
@@ -0,0 +1,14 @@
+const argv = process.argv.slice(2);
+const sumArgv = argv.reduce((a, b) => Number(a) + Number(b));
+const average = sumArgv/ argv.length;
+
+//What should happen if the provided arguments are not numbers 
+if (isNaN(average)){
+ console.log("You have Characters Pls Type Only Number!")
+}
+
+else {
+ console.log(`The Average Of ${sumArgv} Is ${average.toFixed(0)}`);
+}
+
```

Comment:
could you please update warm task. Its seem to look wrong file updated in this Nodejs-week2 branch.